src/fromager/commands/download_sequence.py

```diff
@@ -59,31 +60,26 @@ def download_sequence(
         build_order = json.load(f)
 
     def download_one(entry: dict[str, typing.Any]):
+        req = Requirement(f"{entry['dist']}=={entry['version']}")
+
         if entry["prebuilt"]:
-            logger.info(f"{entry['dist']}: uses a pre-built wheel, skipping")
+            if include_wheels:
+                wheels.download_wheel(req, entry["source_url"], wkctx.wheels_downloads)
+            else:
+                logger.info(f"{entry['dist']}: uses a pre-built wheel, skipping")
             return
 
-        req = Requirement(f"{entry['dist']}=={entry['version']}")
-
-        if entry["source_url_type"] == "sdist":
-            try:
-                source_url, version = sources.resolve_source(
-                    ctx=wkctx, req=req, sdist_server_url=sdist_server_url
-                )
-                sources.download_source(
-                    ctx=wkctx,
-                    req=req,
-                    version=version,
-                    download_url=source_url,
-                )
-            except Exception as err:
-                logger.error(f"failed to download sdist for {req}: {err}")
-                if not ignore_missing_sdists:
-                    raise
-        else:
-            logger.info(
-                f"{entry['dist']}: uses a {entry['source_url_type']} downloader, skipping"
```

Comment:
I think the logic here should be the same if it is not a prebuilt wheel. Apart from "sdist" the only other type we have is "override" in which case we expect that there is some override that will invoked in `download_source` anyways that handles that particular package

Reply:
It was only handling sdists because we expect overrides for download source to be downloading something that is not an sdist file (either a release tarball or cloning a git repository).

Reply:
I see should I revert this then? Although the description of the command says that `Performs the equivalent of the 'step download-source-archive' command for each item in the build order file` and the `step download-source-archive` command would have downloaded non-sdist types as well

Reply:
ahhh I see. Yeah it puts them in the same place. In that case I will add that if check again

Reply:
done